app/src/main/java/com/dkanada/gramophone/adapter/song/SongAdapter.java

```diff
@@ -111,6 +113,15 @@ public void onBindViewHolder(@NonNull final ViewHolder holder, int position) {
             holder.text.setText(getSongText(song));
         }
 
+        Log.d("XYZ", getSongTitle(song) + ": " + (holder.cached != null) + " " + getSongCached(song) );
```

Comment:
Always a fan of more logs but would prefer a proper tag and some string formatting.

Reply:
Oops, forgot to remove that loll